test/core/libp2p/crypto/key_generator_test.cpp

```diff
@@ -269,7 +269,7 @@ TEST_F(GenerateKeyTest, Ed25519KeysNotSame) {
  * @when generateSecp256k1 is called
  * @then obtained key pair have type SECP256K1
  */
-TEST_F(GenerateKeyTest, generateSecp256k1Success) {
+TEST_F(GenerateKeyTest, GenerateSecp256k1Success) {
   auto &&res = keygen_.generateSecp256k1();
   ASSERT_TRUE(res);
   auto &&val = res.value();
```

Comment:
instead of this, write 
```
#include "testutil/outcome.hpp"

EXPECT_OUTCOME_2(val, keygen_.generateSecp256k1());
```

_2 = 2 arguments in macro
_1 = 1 argument in macro

```
// like this
EXPECT_OUTCOME_1(keygen_.generateSecp256k1());
```

Reply:
Ok, refactored, added EXPECT_OUTCOME_FALSE{_1,_2}.